packages/mds-provider/api.ts

```diff
@@ -221,66 +204,56 @@ function api(app: express.Express): express.Express {
     return undefined
   }
 
-  /**
-   * Read Device from cache if possible, else fall through to db
-   * @param  {device_id}
-   * @return {Device}
-   */
-  async function getDevice(device_id: UUID): Promise<Device> {
-    // TODO get device from cache, and if not cache, db
-    return db.readDevice(device_id)
-  }
-
   /**
    * Convert a Telemetry object into a GeoJSON Feature
    * @param item a Telemetry object
    * @returns a GeoJSON feature
    */
-  function asFeature(item: Telemetry): Feature {
-    return {
-      type: 'Feature',
-      properties: {
-        timestamp: item.timestamp
-      },
-      geometry: {
-        type: 'Point',
-        coordinates: [round(item.gps.lng, 6), round(item.gps.lat, 6)]
-      }
-    }
-  }
+  // function asFeature(item: Telemetry): Feature {
```

Comment:
Why commenting out versus nuking/moving to `mds-utils`?

Reply:
They will be making a comeback.